html/themes/custom/common_design_subtheme/css/styles.css

```diff
@@ -67,3 +67,8 @@
   vertical-align: middle;
   background: transparent url("../components/rw-brand/rw-icons-sprite.svg") -84px -108px no-repeat;
 }
+
+.grid--locations .form-checkboxes {
```

Comment:
I had to remove `.grid-locations` to get this to render. We're trying to style an instance of `paragraph--type-email-alerts`, right?

Reply:
We are

Reply:
I've pushed the missing file

Reply:
I'm not a fan of Grid for alphabetical stuff but I'll follow-up later on.